Documentation/cookbook_overview.rst

```diff
@@ -1,10 +1,12 @@
 Overview
 ========
 
-This documentation is a collection of useful dynamic simulation examples to help users build models from the scratch with little efforts. Each example is demonstrated carefully with geometry building, parameter setup and result visualization. Users are suggested to repeat each example in order to get a comprehensive idea of how to set up dynamic simulation models with SeisSol. The updates of source software and news of SeisSol should be posted in SeisSol GitHub (https://github.com/SeisSol/SeisSol) or Readdocs (https://seissol.readthedocs.io/en/latest).
+This documentation is a collection of useful dynamic simulation examples to help users build models from the scratch with little efforts. Each example is demonstrated carefully with geometry building, parameter setup and result visualization. Users are suggested to repeat each example in order to get a comprehensive idea of how to set up dynamic simulation models with SeisSol. The updates of source software and news of SeisSol should be posted in SeisSol GitHub (https://github.com/SeisSol/SeisSol).
 
 SeisSol is a part of SCEC dynamic code validation project (Harris et al. 2018) (http://scecdata.usc.edu/cvws/). Here we show several SCEC benchmarks for beginners to quickly catch up with SeisSol workflow. Each benchmark example is composed of a short problem description, a section of *geometry, initial setups (stress, nucleation, friction, etc.)*, and *simulation results*.
 
+Please note that examples used here are only for demonstration purpose. Therefore, low spatial resolution (200 m) and low intergration order (O4) is used. For detailed benchmark tests please refer to SCEC benchmark center. 
```

Comment:
intergration -> integration. I think the term "integration order" is misleading and which order is low and which is high is up to discussion ;-) I would actually delete the second sentence completely.

Reply:
One could also add a link at this point: http://scecdata.usc.edu/cvws/

Reply:
also 200m is not necessary low
I would rather say that the spatial resolution used may not fully resolve the dynamics